packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -140,22 +145,23 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
             'nt-bg-severity-low-alpha-100 hover:nt-bg-severity-low-alpha-50':
               props.notification.severity === SeverityLevelEnum.LOW,
           }
-        )
-      )}
+        ),
+        context: { notification: props.notification } satisfies Parameters<AppearanceCallback['notification']>[0],
```

Comment:
now we can pass the `context` argument to that will be called with the appearance key callback function